.github/workflows/build-apps.yml

```
@@ -0,0 +1,118 @@
+name: build-apps
+on:
+  push:
+    branches:
+      - 'v2'
```

Commit before merging
```suggestion
      - master
```